tests/test_dataapi.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+import responses
+from learnosity_sdk.request import DataApi
+
+security = {
+    'consumer_key': 'MY_CONSUMER_KEY',
```

Comment:
Does this test even pass with those creds?

We can use the demo creds, I think.

Reply:
The credentials don't matter. Using the `responses` library, the `requests` library is mocked, and so a request is never sent to the server.

Reply:
Ah, right, I missed that it was mocked.

I think we should still put the credentials in. Even if the response it mocked, we want the request to be as real as possible, so we could run it for integration tests, to (which we may actually want to consider).